src/oemof/solph/components/_generic_storage.py

```diff
@@ -1258,7 +1258,7 @@ def _storage_investvar_bound_rule(block, n, p):
         i = {n: [i for i in n.inputs][0] for n in group}
         o = {n: [o for o in n.outputs][0] for n in group}
 
-        reduced_periods_timesteps = [(p, t) for (p, t) in m.TIMEINDEX if t > 0]
+        reduced_periods_timeindex = [(p, t) for (p, t) in m.TIMEINDEX if t > 0]
```

Comment:
To my understanding, this removes all indexes with t <= 0. If we can assume the index to be sorted ans starting with 0, this can be more efficient, sure.

Reply:
Okay, I think, this can be resolved as it was rather a remark, not a demand for a change.